api/.env.example

```diff
@@ -9,6 +9,8 @@ S3_BUCKET="0lfyi-v7"
 S3_STORAGE_CLASS="STANDARD"
 S3_USE_SSL=false
 
+NODE_ENV=local
```

Comment:
you can remove this an assume that if `NODE_ENV !== "production"`, we are not in production mode.